lib/api_test.go

```diff
@@ -0,0 +1,95 @@
+package sebak
+
+import (
+	"boscoin.io/sebak/lib/storage"
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+	"sync"
+	"testing"
+	"time"
+)
+
+func TestStreaming(t *testing.T) {
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	// Setting Server
+	storageConfig, err := sebakstorage.NewConfigFromString("memory://")
+	if err != nil {
+		t.Errorf("failed to initialize file db: %v", err)
+
+	}
+	storage, err := sebakstorage.NewStorage(storageConfig)
+	if err != nil {
+		t.Errorf("failed to initialize file db: %v", err)
+	}
+	defer storage.Close()
+
+	ctx := context.WithValue(context.Background(), "storage", storage)
+	router := mux.NewRouter()
+	router.HandleFunc("/account/{address}", GetAccountHandler(ctx)).Methods("GET")
+	server := &http.Server{Addr: ":5000", Handler: router}
```

Comment:
IHMO, Using `net/http/httptest` can be better for using testing :-D.

Reply:
I tried the httptest.
May I change when I add another APIs?

Reply:
Using httptest is optional :-)!